src/token.ts

```diff
@@ -8,6 +8,10 @@ export interface ISecret {
 }
 
 export interface ISecretsConnector extends IDataConnector<ISecret> {}
+export interface ISecretsConnectorList<T = ISecret> {
```

Comment:
Wondering if this should be named `ISecretsList` instead?

With `ISecretsConnectorList`, on may think this is a connector for interacting with lists of secrets at once?